script.js

```diff
@@ -1,4 +1,215 @@
+    return drawCardMode()
+  }
+  else if(GAMEMODE == 'ace'){
+    return aceChose(input)
+  }
+   else if(GAMEMODE == 'hit or stand'){
+    return hitOrStand(input)
+  }
+   else if(GAMEMODE == 'win or lose'){
+    return winOrLose()
+  }
 };
+
+var wager = function(){
+  return `you bet ${betNumber} point`
+}
+
+var drawCardMode = function(){
+  var playerCard = deck.pop()
+  var dealerCard = deck.pop()
+  playerTotal += playerCard.rank
+  dealerTotal += dealerCard.rank
+  if(playerCard.rank == 1){
+      GAMEMODE = 'ace'
+      playerTotal -= playerCard.rank
+      return `${username} card : ${playerCard.name} of ${playerCard.suit}<br>dealer card : ${dealerCard.name} of ${dealerCard.suit}<br> your current total is ${playerTotal}, you got ace input '1' or '11'`
+  }
+  return `${username} card : ${playerCard.name} of ${playerCard.suit} <br>dealer card : ${dealerCard.name} of ${dealerCard.suit}<br>click submit to add the card or input "stand" if you are done`
+}
+
+var hitOrStand = function(input){
+  if(input == ''){
+    var playerCard = deck.pop()
+    playerTotal += playerCard.rank
+    if(playerCard.rank == 1){
+      GAMEMODE = 'ace'
+      playerTotal -= playerCard.rank
+      return `you got ${playerCard.name} of ${playerCard.suit}<br> your current total is ${playerTotal}, input '1' or '11'`
+    }
+    if(playerTotal >21){
+    GAMEMODE = 'bet'
+    playerPoint -= betNumber
+    dealerPoint += betNumber
+    return `you got ${playerCard.name} of ${playerCard.suit}<br>your total is ${playerTotal}<br>bustedd!<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+    }
+    else{
+      return `you got ${playerCard.name} of ${playerCard.suit}<br>your total is ${playerTotal}<br>click submit to add the card or input "stand" if you are done`
+    }
+  }
+    GAMEMODE = 'win or lose'
+    return `your total is ${playerTotal}, click submit to decide the winner`
+}
+
+var aceChose = function(input){
+  if(input == '1'){
+    playerTotal +=1
+    if(playerTotal >21){
+    GAMEMODE = 'bet'
+    return `you chose 1<br>your total is ${playerTotal}<br>bustedd!<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+    }
+    GAMEMODE = 'hit or stand'
+    return `you chose 1<br>your total is ${playerTotal}<br>click submit to add the card or input "stand" if you are done`
+  }
+  else if(input == '11'){
+    playerTotal +=11
+    if(playerTotal >21){
+    GAMEMODE = `bet`
+    return `you chose 11<br>your total is ${playerTotal}<br>bustedd!<br><br>${username} point :${playerPoint}<br>dealer point :${dealerPoint}<br><br>enter number of point you want to bet`
+    }
+    GAMEMODE = 'hit or stand'
+    return `you chose 11<br>your total is ${playerTotal}<br>click submit to add the card or input "stand" if you are done`
+  }
+  else{
+    return `your current total is ${playerTotal}, you got ace input '1' or '11'`
+  }
+
```

Comment:
since you have quite a few cases, it might be worthwhile to consider using a switch statement here instead.